test/verify/check-sosreport

```diff
@@ -60,9 +61,21 @@ threads=1
         # while the download is ongoing, it will have an *.xz.tmpsuffix name, gets renamed to *.xz when done
         wait(lambda: len(glob.glob(os.path.join(b.cdp.download_dir, "sosreport-*.xz"))) > 0)
         report = glob.glob(os.path.join(b.cdp.download_dir, "sosreport-*.xz"))[0]
+
         # Check that /etc/release was saved. It the files does not exist, getmember raises KeyError
-        with tarfile.open(report) as tar:
-            tar.getmember(os.path.join(tar.getnames()[0], "etc/os-release"))
+        # Sometimes it takes a bit of time until the file can be opened. Try it 3 times.
```

Comment:
That's a bit weird. Usually browsers append some kind of .tmp suffix, and that's exactly what the wait() waits for above. So from the browser's POV the download should be okay. Too bad that the tarfile error message is rather useless. Well *shrug*, this isn't too bad, let's go with that.